colcon_core/shell/template/prefix_util.py.em

```diff
@@ -42,19 +48,22 @@ def main(argv=sys.argv[1:]):  # noqa: D103
     packages = get_packages(Path(__file__).parent, args.merged_install)
 
     ordered_packages = order_packages(packages)
+    commands = []
```

Comment:
This change isn't necessary. Please just print the lines returned by `_remove_trailing_separators()`.

Reply:
Done dd07358